test_requirements.txt

```diff
@@ -2,4 +2,5 @@ coverage==3.7
 nose==1.3.0
 mock==1.0.1
 pep8==1.7.0
+requests==2.10.0
```

Comment:
... and then @jmbowman discovered that it's not even being used in the test file that imported it so we could get rid of it anyhow. Bonus!